lib/Models/Terria.ts

```diff
@@ -935,6 +935,7 @@ export default class Terria {
   addShareKey(id: string, shareKey: string): void {
     if (id === shareKey || this.shareKeysMap.has(shareKey)) return;
     this.shareKeysMap.set(shareKey, id);
+    // eslint-disable-next-line @typescript-eslint/no-unused-expressions
```

Comment:
These `x ?? y` constructs are permitted with ESLint 9.17, so the directive will disappear when upgrading ESLint.